### DISCUSSION TITLE
7회차 출석 여부 및 발전 기금 현황

### DISCUSSION THREAD

**Discussion by BvrPark** [cowaine/spring-boot-for-msa#65] [0. Announcements]
## 💻 출석 및 발전 기금 현황

|                    Username                     | 7회차 출석 |발전 기금 현황 | PR부재 | Q&A부재 | 책무 불이행 | 지각 | 결석 |
|:-----------------------------------------------:|:----:|:----:|:---:|:---:|:--:|:--:|:--:|
|  [ahngilwoong](https://github.com/ahngilwoong)  | ✅  |0원 | 0회 | 0회 | 0회 | 0회 | 0회 |
|      [coalong](https://github.com/coalong)     | ✅  | 0원 | 0회 | 0회 | 0회 | 0회 | 0회 |
|       [CoRock](https://github.com/corock)      | ✅  | 0원 | 0회 | 0회 | 0회 | 0회 | 0회 |
|      [BvrPark](https://github.com/BvrPark)      | ✅  | 0원 | 0회 | 0회 | 0회 | 0회 | 0회 |
|    [KIrisation](https://github.com/bunsung92)    | ✅  | 0원 | 0회 | 0회 | 0회 | 0회 | 0회 |
|       [JoisFe](https://github.com/joisfe)      | ✅  | 0원 | 0회 | 0회 | 0회 | 0회 | 0회 |
| [Youngju Jang](https://github.com/youngju-jang) | ✅  | 0원 | 0회 | 0회 | 0회 | 0회 | 0회 |

</br>

### ❗️ 발전 기금 부과 방식(1회당)
- PR부재 : 5000원
- Q&A부재 : 5000원
- 책무 불이행 : 5000원
- 지각 : 5000원
- 결석 : 10000원

**Comment by Irisation23** [comment]
기존 아이디 `ding_cook` -> `KIrisation` 으로 변경 했습니다.
노고에 감사드립니다.
